packages/site-parsers/src/utils/logger.js

```diff
@@ -0,0 +1,29 @@
+const debugLog = require( 'debug' );
+
+function Logger( provider, debug ) {
```

Comment:
Since debug output is [triggered through other external means](https://www.npmjs.com/package/debug) (for example an environment variable), it's no longer necessary to provide the flag:
```suggestion
function Logger( provider ) {
```